### DISCUSSION TITLE
Chip unavailable icon color

### DISCUSSION THREAD

**Discussion by pedolsky** [piitaya/lovelace-mushroom#1036] [Q&A]
Hi there,

just realised that my entity and light chips doesn't reflect unavailable state in their icon color. Has it always been like this?
Any friendly answer is appreciated!

**Comment by piitaya** [comment]
Yes, the disabled/unavailable color for chip was always black. May be this can be improved 🙂 Mushroom should use [HA state colors](https://www.home-assistant.io/integrations/frontend/#state-color) but it's a huge breaking change 😅

**Reply by pedolsky** [reply]
Thank you very much for your quick reply. Should I start a PR or enhancement or is this something that is still too far in the future?

**Reply by piitaya** [reply]
As a quick fix, we can use disabled color for unavailable state.
